events/game_rule_events.txt

```diff
@@ -6739,1132 +6739,1289 @@ character_event = { # Delayed cleansing
 	}
 }
 
-# character_event = { # Generate 'Fake Family' for characters without any
-	# id = GR.30
+character_event = { # Generate 'Fake Family' for characters without any
+	id = GR.30
 	
-	# hide_window = yes
+	hide_window = yes
+	
+	is_triggered_only = yes
 	
-	# is_triggered_only = yes
+	only_playable = yes
+	min_age = 16
 	
-	# only_playable = yes
-	# min_age = 16
+	# Warcraft
 	# only_men = yes
 	
-	# trigger = {
-		# OR = {
-			# AND = {
-				# ai = yes
-				# has_game_rule = {
-					# name = generate_families
-					# value = ai_only
-				# }
-			# }
-			# has_game_rule = {
-				# name = generate_families
-				# value = on
-			# } 
-		# }
-		# is_save_game = no
-		# OR = {
-			# is_married = no
-			# AND = {
-				# is_married = yes
-				# num_of_children < 1
-				# spouse = {
-					# is_lowborn = yes
-				# }
-			# }
-		# }
-		# mercenary = no
-    	# holy_order = no
-    	# is_landed = yes
-		# OR = {
-			# is_feudal = yes
-			# is_nomadic = yes
-			# is_tribal = yes
-		# }
-		# NOR = {
-			# government = confucian_bureaucracy
-			# government = order_government
-			# trait = eunuch
-			# trait = celibate
-			# has_nickname = nick_the_chaste
-		# }
-		# OR = {
-			# NOT = { num_of_dynasty_members = 2 }
-			# #NOT = {
-			# #	any_dynasty_member = {
-			# #		NOR = {
-			# #			character = ROOT
-			# #			is_child_of = ROOT
-			# #		}
-			# #		is_alive = yes
-			# #	}
-			# #}
-			# NOT = {
-				# any_close_relative = {
-					# NOR = {
-						# character = ROOT
-						# is_child_of = ROOT
-					# }
-					# is_alive = yes
-				# }
-			# }
-			# AND = {
-				# num_of_children >= 1
-				# NOT = {
-					# any_child = {
-						# mother = { always = yes }
-					# }
-				# }
-				# NOT = {
-					# any_dynasty_member = {
-						# NOT = {
-							# ROOT = {
-								# is_father = PREV
-							# }
-						# }
-					# }
-				# }
-			# }
-		# }
-	# }
+	trigger = {
+		OR = {
+			# Warcraft
+			AND = {
+				trait = infertile
+				has_game_rule = {
+					name = generate_families
+					value = infertile
+				}
+			}
+			
+			AND = {
+				ai = yes
+				has_game_rule = {
+					name = generate_families
+					value = ai_only
+				}
+			}
+			has_game_rule = {
+				name = generate_families
+				value = on
+			} 
+		}
+		is_save_game = no
+		OR = {
+			is_married = no
+			AND = {
+				is_married = yes
+				num_of_children < 1
+				spouse = {
+					is_lowborn = yes
+				}
+			}
+		}
+		mercenary = no
+    	holy_order = no
+    	is_landed = yes
+		OR = {
+			is_feudal = yes
+			is_nomadic = yes
+			is_tribal = yes
+		}
+		NOR = {
+			government = confucian_bureaucracy
```

Comment:
There is no confucian_bureaucracy in mod, but ok

Reply:
Purged!